### DISCUSSION TITLE
Suggestion for New Permission

### DISCUSSION THREAD

**Discussion by Mr-Atanu-Roy** [encode/django-rest-framework#9152] [Ideas & Suggestions]
The permissions provided by DRF are very good and meets most of the use-cases of developers.

But with my developing experience I have notice that if there was a permission where the **owner of a model object is only allowed to access the object** then it will be a very useful one. There are permissions like **DjangoModelPermissions** and **DjangoObjectPermissions** which are some what similar to my suggestion. But if we look at this it would be more specific.

So I want to contribute to the DRF community by adding this permission.

**Comment by lovelydinosaur** [comment]
The DRF project isn't accepting new functionality at this point in the project lifecycle.
